lib/generate_functions.py

```diff
@@ -360,6 +369,7 @@ def parse_header(header_name: str, output_file: str, ext_file: str, prefix: str,
 
         if return_cast:
             zig_funcs.append(
+                (inline_comment if inline_comment != "" else "") +
```

Comment:
This if is not required, just `inline_comment` will do

Reply:
Ah yes, I forgot about this. I added it before I created the little "sanity check" if statement found earlier